src/tokens/letterSpacing.js

```diff
@@ -0,0 +1,14 @@
+const light = core => ({
+  SmallUppercase: core.LetterSpacing.Uppercase,
```

Comment:
Are we planning on having different values for these in the future?

Reply:
Possibly

Reply:
Do we need more than one then?

Reply:
Well we have 3 different uppercase styles (Tiny, XSmall, and Small). 

We might adjust values in the future. I'm not really sure.